source/release-notes/1.2.txt

```diff
@@ -17,14 +17,20 @@ Release Notes for ``mongosync`` 1.2
 Migrating Older Versions of MongoDB
 -------------------------------------
 
-Starting in version 1.2, ``mongosync`` can sync between clusters running 
-some older (pre-6.0) releases of MongoDB Server. This feature requires
-additional preparation and configuration in pre-6.0 releases.
+Starting in version 1.2, ``mongosync`` can sync between clusters running
+some pre-6.0 releases of MongoDB Server for one time migration onto
```

Comment:
Can we remove "some" here or add some additional context to what pre-6.0 releases are supported? 
If it is a specific minor release or range of releases consider adding the minor releases in parenthesis.

```suggestion 
   pre-6.0 releases of MongoDB Server for one time migration onto
```

Reply:
Done! I removed "some" and talked to @ajayvijayakumar123 who said that engineering does not want to publicly document which versions prior to 6.0 are supported (yet).